spec/models/agent_log_spec.rb

```diff
@@ -11,7 +11,7 @@
     it "requires an agent" do
       @log.agent = nil
       expect(@log).not_to be_valid
-      expect(@log).to have(1).error_on(:agent)
+      expect(@log).to have_at_least(1).error_on(:agent)
```

Comment:
Without DJ two errors are returned, one because the `belong_to` association is not optional and one for the presence validation.